test/src/test/java/hudson/model/UpdateSiteTest.java

```diff
@@ -205,6 +206,19 @@ public void shutdownWebserver() throws Exception {
         assertNotEquals("plugin data is present", Collections.emptyMap(), site.getData().plugins);
     }
 
+    @Issue("JENKINS-73760")
+    @Test
+    public void isLegacyDefault() {
+        assertFalse("isLegacyDefault should be false with null id", new UpdateSite(null, "url").isLegacyDefault());
+        assertFalse(
+                "isLegacyDefault should be false when id is not default and url is http://updates.jenkins-ci.org/",
+                new UpdateSite("dummy", "http://updates.jenkins-ci.org/").isLegacyDefault());
+        assertTrue(
+                "isLegacyDefault should be true when id is default and url is http://updates.jenkins-ci.org/",
+                new UpdateSite(UpdateCenter.PREDEFINED_UPDATE_SITE_ID, "http://updates.jenkins-ci.org/").isLegacyDefault());
+        assertFalse("isLegacyDefault should be false with null url", new UpdateSite(null, null).isLegacyDefault());
+    }
+
```

Comment:
Adapting the pre-#6116 code to handle the migration from http://updates.jenkins-ci.org/ to https://updates.jenkins.io/ rather than the migration from http://hudson-ci.org to http://updates.jenkins-ci.org/